docs/README.md

```diff
@@ -29,6 +29,7 @@ The LogDNA agent is a resource-efficient log collection client that forwards log
   * [Configuring Lookback](#configuring-lookback)
   * [Configuring Journald](#configuring-journald)
   * [Configuring Kubernetes Events](#configuring-events)
+  * [Configuring regex for redaction and exclusion or inclusion](#configuring-regex-for-redaction-and-exclusion-or-inclusion)
```

Comment:
The anchor needs updating :)

Reply:
I wasn't sure how to handle the special character (/) in the title...so I just changed the heading to match this link in ToC. But we can make heading whatever we want, let me know if we need to tweak.